examples/cli.rs

```diff
@@ -0,0 +1,120 @@
+// A basic client example.
```

Comment:
This exists here: https://github.com/cloudflare/odoh-client-rs

Reply:
This is just an example, not mean to replace the client program.